shared-scripts.sh

```diff
@@ -164,6 +164,14 @@ function _publishToLocalRegistry {
 
     _generateChangeLog
 }
+function _publishLocalWorkspace {
```

Comment:
Do we want to expose a top level script in the `package.json` to be able to easily run this? Like `yarn publish-to-local`?

Reply:
I went back & forth on this and landed on "not right now." I want to see it in action a few times first, and then we can make sure that a yarn task properly encompasses all the steps. For now, it's not too burdensome; involving a couple of bash invocations rather than a yarn task.